docs/running.md

```diff
@@ -21,6 +21,8 @@ Or as a [Docker](https://www.docker.com/) container:
 ```bash
 sbt docker:publishLocal
 
+apk add git # base image is alpine and git is called from scala-steward
```

Comment:
Shouldn't we add this to https://github.com/fthomas/scala-steward/blob/77925ac9fe5fdb46a32273cb83054875f4e31e03/build.sbt#L178-L184?

Reply:
yes ... and more ! I'm going to give you the needed steps in a while (unless my boss joins the meeting we are supposed to have ^^)

Reply:
I'm running this set of commands from the docker image to install scala + sbt:

```
SCALA_HOME=/usr/share/scala
SCALA_VERSION=2.12.11
SBT_VERSION=1.3.8

apk add  wget git ca-certificates  &&     cd "/tmp" &&     wget --no-verbose "https://downloads.typesafe.com/scala/${SCALA_VERSION}/scala-${SCALA_VERSION}.tgz" &&     tar xzf "scala-${SCALA_VERSION}.tgz" &&     mkdir "${SCALA_HOME}" &&     rm "/tmp/scala-${SCALA_VERSION}/bin/"*.bat &&     mv "/tmp/scala-${SCALA_VERSION}/bin" "/tmp/scala-${SCALA_VERSION}/lib" "${SCALA_HOME}" &&     ln -s "${SCALA_HOME}/bin/"* "/usr/bin/" &&     apk del .build-dependencies &&     rm -rf "/tmp/"*

export PATH="/usr/local/sbt/bin:$PATH" &&  apk update && apk add ca-certificates wget tar && mkdir -p "/usr/local/sbt" && wget -qO - https://piccolo.link/sbt-${SBT_VERSION}.tgz | tar xz -C /usr/local/sbt --strip-components=1 && sbt sbtVersion
```

I'm still wondering if we shouldn't switch the base image to `hseeberger/scala-sbt:11.0.6_1.3.8_2.12.10` instead of handling all this stuff in scala-steward ?

Reply:
with the previous set of commands, when I run scala-steward I now have a very strange behaviour (nothing done, but no errors ...)

Sorry, I need to switch to meetings, hope I brought a little help ...

```
/opt/docker/bin/scala-steward -DROOT_LOG_LEVEL=INFO -DLOG_LEVEL=INFO --disable-sandbox --process-timeout 30min --do-not-fork --env-var SBT_OPTS=-XX:MaxMetaspaceSize=512m --workspace /app/workspace/ --repos-file /app/repos-colisweb-opensource.md --git-ask-pass /app/pass-colisweb.sh --git-author-name 'Scala Steward' --git-author-email gitlab@colisweb.com --vcs-type gitlab --vcs-api-host https://gitlab.com/api/v4/ --vcs-login gitlab.ci1
2020-03-26 09:15:20,668 INFO
  ____            _         ____  _                             _
 / ___|  ___ __ _| | __ _  / ___|| |_ _____      ____ _ _ __ __| |
 \___ \ / __/ _` | |/ _` | \___ \| __/ _ \ \ /\ / / _` | '__/ _` |
  ___) | (_| (_| | | (_| |  ___) | ||  __/\ V  V / (_| | | | (_| |
 |____/ \___\__,_|_|\__,_| |____/ \__\___| \_/\_/ \__,_|_|  \__,_|
 v0.5.0-300-77925ac9-SNAPSHOT

2020-03-26 09:15:20,686 INFO  Run self checks
2020-03-26 09:15:23,098 INFO  Add global sbt plugins
2020-03-26 09:15:23,177 INFO  Clean workspace /app/workspace
2020-03-26 09:15:26,886 INFO  ──────────── Steward colisweb-open-source/scala/skills ────────────
2020-03-26 09:15:26,901 INFO  Check cache of colisweb-open-source/scala/skills
2020-03-26 09:15:27,030 INFO  Skipping due to previous error
2020-03-26 09:15:27,093 INFO  ──────────── Total time: Steward colisweb-open-source/scala/skills: 202ms ────────────
2020-03-26 09:15:27,095 INFO  ──────────── Steward colisweb-open-source/scala/JRubysnessHours ────────────
2020-03-26 09:15:27,095 INFO  Check cache of colisweb-open-source/scala/JRubysnessHours
2020-03-26 09:15:27,124 INFO  Skipping due to previous error
2020-03-26 09:15:27,167 INFO  ──────────── Total time: Steward colisweb-open-source/scala/JRubysnessHours: 72ms ────────────
2020-03-26 09:15:27,168 INFO  ──────────── Steward colisweb-open-source/scala/scala-distances ────────────
2020-03-26 09:15:27,168 INFO  Check cache of colisweb-open-source/scala/scala-distances
2020-03-26 09:15:28,864 INFO  Refresh cache of colisweb-open-source/scala/scala-distances
```